utils/src/main/java/com/cloud/utils/PropertiesUtil.java

```diff
@@ -34,6 +34,10 @@
 public class PropertiesUtil {
     private static final Logger s_logger = Logger.getLogger(PropertiesUtil.class);
 
+    public static String getDefaultApiCommandsFileName() {
+        return "commands.properties";
+    }
```

Comment:
commands.properties is consumed by server, plugins/acl/static-role-based and engine/schema packages. I'm keeping it this way so it can be consumed by all these packages, as utils is used by all and PropertiesUtils is a utility class to manage .properties files